src/app.md

```diff
@@ -233,28 +233,35 @@ fn main() {
 <a name="ex-file-predicate"></a>
 ##  Recursively find all files with given predicate
 
-Use the file names and metadata to recursively find in the current directory JSON files modified within the last day.
-
 [![walkdir-badge]][walkdir] [![cat-filesystem-badge]][cat-filesystem]
 
+Use the file names and metadata to recursively find in the current directory 
```

Comment:
>Use  file names and metadata

The fact of using "file names" and "metadata" is less important than the implementations behind it. I would suggest replacing these terms with links to the used methods / structs in crate docs.